### PR TITLE
[Packager] English edits

### PR DESCRIPTION
**Pull request by jillnogold**

### CODE REVIEW

**Comment by guy1992l** [comment]
Problem here is you have changed error messages that are being validated for being raised in tests. You need to change them in the failed test or roll back the error edits.

**Review by guy1992l — CHANGES_REQUESTED** [review]

File: mlrun/package/packagers/default_packager.py [hunk 1]
```diff
@@ -169,86 +169,86 @@ def get_default_artifact_type(pack_or_unpack: str) -> str:
...
               pass
 
       Where 'x' is the artifact type, 'obj' is the object to pack, `key` is the key to name the artifact and `...` are
-      additional custom log hint configurations and the returning values are the packed artifact and the instructions
-      for unpacking it, or in case of result, the dictionary of the result with its key and value. The log hint
-      configurations are sent by the user and shouldn't be mandatory, meaning they should have a default value
-      (otherwise, the user will have to add them to every log hint).
+      additional, custom, log hint configurations. The returned values are the packed artifact and the instructions
+      for unpacking it, or in the case of result, the dictionary of the result with its key and value. The log hint
+      configurations are sent by the user and are mandatory. If unspecified, they get a default value.
```
[the remark this answers is not in the document]
Comment (jillnogold) [reply mlrun/package/packagers/default_packager.py:199]: @guy1992l How does the user set a default value? If we want to say it should have a default - think we should explain that.

File: docs/api/mlrun.package.rst [hunk 2]
```diff
@@ -3,16 +3,14 @@
 mlrun.package
 =============
 
-MLRun package enable **fully automated experiment and pipeline tracking and reproducibility**, easy
-**passing python objects between remote jobs** while **not requiring any form of editing** to the actual function
-original code. Simply set the function code in a project and run, MLRun will take care of the dirty work.
+MLRun package enables fully-automated experiment and pipeline tracking, with reproducibility, and easy
+passing of python objects between remote jobs, without requiring any form of editing to the function's
+original code. The packagers are defined in the function code of a project.
```
[the remark this answers is not in the document]
Comment (jillnogold) [reply docs/api/mlrun.package.rst]: reverted to your wording

**Review by guy1992l — APPROVED** [review]
:)

**Pull request merged**